<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtameshiHistUI</class>
 <widget class="QMainWindow" name="OtameshiHistUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>737</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OtameshiHistUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1061</width>
      <height>741</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QGraphicsView" name="graphicsReference">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>250</y>
        <width>400</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>200</height>
       </size>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkCVImage">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>660</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>0</height>
       </size>
      </property>
      <property name="text">
       <string>CVImage</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>80</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>#Ball_histogram</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdScoreEuclid">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>460</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
      <property name="smallDecimalPoint">
       <bool>false</bool>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdScoreNormalize">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>460</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="labeldebug">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>640</y>
        <width>27</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>State</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboImageName">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>660</y>
        <width>400</width>
        <height>20</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>20</height>
       </size>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>142.png</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>143.png</string>
       </property>
      </item>
     </widget>
     <widget class="QGraphicsView" name="graphicsColorHist">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>460</y>
        <width>400</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>200</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>442</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>■Euclid_distance</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdScoreColorHist">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>460</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdScoreCosine">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>460</y>
        <width>150</width>
        <height>150</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>150</width>
        <height>150</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>440</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>■Normalize_expression</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushStar">
      <property name="geometry">
       <rect>
        <x>1143</x>
        <y>752</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string>★</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="Chisato">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>660</y>
        <width>78</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>    Chisato</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsBallHist">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>30</y>
        <width>400</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>200</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>440</y>
        <width>105</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>■Cosine_correlation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>430</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>■Color_histogram</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="visualizeColor">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>631</width>
        <height>421</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>421</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>230</y>
        <width>112</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>#Reference_histogram</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
